src/java.net.http/share/classes/jdk/internal/net/http/Exchange.java

```diff
@@ -111,8 +120,12 @@ HttpClientImpl client() {
 
     // Keeps track of the underlying connection when establishing an HTTP/2
     // exchange so that it can be aborted/timed out mid setup.
-    static final class ConnectionAborter {
+    final class ConnectionAborter {
+        // In case of HTTP/3 direct connection we may have
```

Comment:
"direct" is usually used to refer to connections that don't use a proxy; since H3 connections never use a proxy, can we drop the "direct" here?